Auth0/SafariWebAuth.swift

```diff
@@ -181,6 +181,7 @@ class SafariWebAuth: WebAuth {
 
         entries.forEach { items.append(URLQueryItem(name: $0, value: $1)) }
         components.queryItems = self.telemetry.queryItemsWithTelemetry(queryItems: items)
+        components.percentEncodedQuery = components.percentEncodedQuery?.replacingOccurrences(of: "+", with: "%2B")
```

Comment:
I had a look at encoding at the key value. However, then the URL will end up double encoded. So this also appears to be the okay by apple http://www.openradar.me/24076063

Reply:
Ugh. Thanks for the link